src/components/TranslationValueRow/index.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import React, { Component } from 'react'
+import { type Locale } from '@types/Locale'
+import { type TranslationValue } from '@types/TranslationValue'
+import { Wrapper, ContentWrapper, LocaleWrapper, Textarea } from './styles'
+
+type Props = {
+  locale: Locale,
+  translationKeyValue: TranslationValue | null,
+  onBlur: string => any
+}
+
+type State = {
+  isFocused: boolean,
+  value: string
+}
+
+class TranslationValueRow extends Component<Props, State> {
+  state = {
+    isFocused: false,
+    value: ''
+  }
+
+  onChange = (value: string) => {
+    this.setState(state => ({
+      ...state,
```

Comment:
`this.setState(state => ({ value }))` work too… :p
`this.setState({ value })` since you don’t need the previous state value 🤷‍♂️